client/view/workbook/sheet/error/Error.html

```diff
@@ -0,0 +1,62 @@
+<div class="container">
+
+
+	<header>
+		<h2>Error</h2>
+	</header>
+
+
+	<main>
+		We couldn't load any rows from this sheet!  Make sure it has rows, and there aren't any blank rows.
+		<div class="try-again">
+				<a href="{ asr.makePath('workbook', { key: key }) }">Try Again</a>
+				•
+				<a href="https://docs.google.com/spreadsheets/d/{key}" target="_blank">Edit Sheet</a>
+		</div>
+	</main>
+
+
+	<footer>
+		<a
+			href="{ asr.makePath('index') }"
+			class=footer-left
+		>
+			Home
+		</a>
+		<a
+			href="{ asr.makePath('workbook.info', { key: key }) }"
+			class="footer-right">
+			Sheet&nbsp;Info
```

Comment:
Does this actually need to be a non-breaking space?

Reply:
If I take out the non-breaking space, it will span two lines, which I think looks a bit funny.

Reply:
Why does it span two lines?  What is it that forces the width of the link to be so small that it wraps?  I'd rather add a general rule to prevent the narrowness/wrapping, if possible.

Reply:
It appears that "Sheet Info" is wider than the 64px set at `flex-basis` on line 60.  When the text does not break, it just overflows it's container.